test/test_helper.rb

```diff
@@ -42,6 +42,11 @@ def assert_changed(object, attribute, &block)
   end
 end
 
+class ActionDispatch::IntegrationTest
+  setup { host! Gemcutter::HOST }
+end
+Capybara.app_host = "http://#{Gemcutter::HOST}"
+
```

Comment:
now that we use `Redirector` on tests, we need to make sure the rack-test HOST is the right one and not `www.example.com` which is the default.